src/utils/extractOptions.js

```diff
@@ -0,0 +1,12 @@
+const extractOptions = data => {
+  var options = [];
+  data.data.forEach((element, i) => {
```

Comment:
also `forEach` should be avoided

try a more functional approach with `map`

Reply:
7f301a73a71fe172d902906517edf5f97374d9d7v